interfaces/cython/cantera/test/test_composite.py

```diff
@@ -148,84 +251,86 @@ def check(a, b, atol=None):
             X /= X.sum()
             a.append(T=T, P=P, X=X)
 
-        data, labels = a.collect_data()
+        data = a.collect_data()
 
         # basic restore
         b = ct.SolutionArray(self.gas)
-        b.restore_data(data, labels)
+        b.restore_data(data)
         check(a, b)
 
         # skip concentrations
         b = ct.SolutionArray(self.gas)
-        b.restore_data(data[:, :2], labels[:2])
+        b.restore_data(OrderedDict([tup for i, tup in enumerate(data.items())
+                                    if i < 2]))
```

Comment:
Wouldn't it be a little more clear to just write:
```suggestion
        b.restore_data({'T': data['T'], 'density': data['density']})
```

Reply:
Will have to look into this. All of this is just because of pesky Xenial and Python 3.5, which doesn’t preserve order of dictionary items. After that, regular dictionaries are sufficient. I suppose Python 3.5 will be supported for some more time ...

Reply:
Alright, I see the point. I guess using a regular dictionary here shouldn't matter anyways, as order won't matter in this case.